### DISCUSSION TITLE
Deployment Firebase error - current domain is not authorized for OAuth operations

### DISCUSSION THREAD

**Discussion by nashvegasnate** [nss-evening-cohort-14/evening-client-side#190] [Look at what I learned!]
Issue:
When trying to login, the pop-up window seems to get stuck and never authorizes user.

Error message:
Info: The current domain is not authorized for OAuth operations. This will prevent signInWithPopup, signInWithRedirect, linkWithPopup and linkWithRedirect from working. Add your domain (deploy-preview-19--XXXXXX.netlify.app) to the OAuth redirect domains list in the Firebase console -> Auth section -> Sign in method tab.

Solution:
On authentication, go to Sign In Method. 
Scroll down to Authorized Domains. 
Click "Add Domain" button. 
Paste the link to the deployed site.

**Comment by drteresavasquez** [comment]
Also see the other tickets on this topic:
- https://github.com/nss-evening-cohort-14/evening-client-side/discussions/132
- https://github.com/nss-evening-cohort-14/evening-client-side/discussions/184

**Comment by zanderspf** [comment]
thank you
